hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/fs/DedicatedDiskSpaceUsage.java

```diff
@@ -51,7 +51,7 @@ public long getUsedSpace() {
    * @return used space
    */
   private long calculateUsedSpace() {
-    return getCapacity() - getFile().getFreeSpace();
+    return getCapacity() - getFile().getUsableSpace();
```

Comment:
For the record: javadoc on `calculateUsedSpace` still mentions "free space".  I guess we can update it in any patches touching this code, no need for a new PR just for this.

Also: `getAvailble()` also returns `getUsableSpace()`, so we could clarify the logic by changing to:

`getCapacity() - getAvailable()`